app/retail/emails.py

```diff
@@ -558,34 +558,34 @@ def render_new_bounty_roundup(to_email):
 </p>
 '''
     highlights = [{
-        'who': 'StevenJNPearce',
+        'who': 'dryajov',
         'who_link': True,
-        'what': 'Added browser reload on file change to Embark!',
-        'link': 'https://gitcoin.co/issue/embark-framework/embark/832/1214',
+        'what': 'Completed one of the largest bounties of all time on MetaMask!',
+        'link': 'https://gitcoin.co/issue/MetaMask/mustekala/21/1279',
         'link_copy': 'View more',
     }, {
-        'who': 'iamonuwa',
+        'who': 'mul1sh',
         'who_link': True,
-        'what': 'Alerted users of the Ethereum network they are using for Electric Feel.',
-        'link': 'https://gitcoin.co/issue/electricfeelco/travay/11/1219',
+        'what': 'Completed his first bounty! Congrats.',
+        'link': 'https://gitcoin.co/issue/diadata-org/coindata/1/1259',
         'link_copy': 'View more',
     }, {
-        'who': 'lastmjs',
+        'who': 'zachzundel',
         'who_link': True,
-        'what': 'Helped grow the Ethereum Alarm Clock!',
-        'link': 'https://gitcoin.co/issue/ethereum-alarm-clock/timenode-core/194/979',
+        'what': 'Moving Sharding forward with Prysmatic Labs!',
+        'link': 'https://gitcoin.co/issue/prysmaticlabs/prysm/497/1212',
         'link_copy': 'View more',
     }, ]
 
     bounties_spec = [{
-        'url': 'https://github.com/tkstanczak/nethermind/issues/86',
-        'primer': 'Implement Discovery on Nethermind.',
+        'url': 'https://github.com/ethereum/EIPs/issues/1442',
+        'primer': 'Document JSON-RPC interface in an EIP.',
     }, {
-        'url': 'https://github.com/PolymathNetwork/polymath-core/issues/272',
-        'primer': 'Work on Automated blacklisting from Polymath.',
+        'url': 'https://github.com/NethermindEth/nethermind/issues/86',
```

Comment:
unfortunately. looks like this one didnt render... http://bits.owocki.com/f9d1ea407fd5/Screen%20Shot%202018-10-01%20at%2012.39.12%20PM.png